api/basic.yaml

```diff
@@ -11,10 +11,18 @@ info:
     email: snlab.org@gmail.com
 paths: {}
 definitions:
+  ResourceID:
+    pattern: ^[0-9a-zA-Z-:@_\.]{1,64}$
+    type: string
+  CostMetric:
+    pattern: ^[0-9a-zA-Z-:@_\.]{1,64}$
```

Comment:
If you check [section_ 10.6 of RFC7285](https://tools.ietf.org/html/rfc7285#section-10.6), you will find the character `@` is not valid here.

Please add the reference to your modification into the commit message.